pyqt-apps/siriushla/as_ap_injection/main.py

```diff
@@ -491,10 +491,10 @@ def _setupTopUpModeWidget(self):
         self._lb_tupustd = SiriusLedState(
             self, pvname.substitute(propty_suffix='Sts'))
 
-        self._ld_tulistd = QLabel('LI Standby', self)
-        pvname = self._inj_prefix.substitute(propty='TopUpLIStandbyEnbl-Sel')
```

Comment:
@anacso17 , are all injection PVs registered in the archiver appliance?

Reply:
Hi @xresende , yes! Those new ones will be renamed and added as well.